openapi-issuer.yaml

```diff
@@ -108,6 +108,31 @@ paths:
           $ref: "#/components/responses/NotFound"
         default:
           $ref: "#/components/responses/Unexpected"
+    patch:
```

Comment:
Thinkins about this, I'd rather use another endpoint` /signature-requests/{id}/status`
and use a `PUT` accepting only the new status as payload (not the whole request).

that's because you're not patching the whole object here, but you're limiting the options to just one field.

Reply:
I was looking at the [Zalando guidelines](https://opensource.zalando.com/restful-api-guidelines/#patch), and i used PATCH just to change only part of the resource and not the whole request (`PUT`).

Do you think maybe it is better to add `PUT` on  `/signature-requests/{id}/status` for a better comprehension?

Reply:
Using `PATCH` is a feasible option if caller can change anything about the resources (moreover Zalando guidelines suggest to use a specific format to do that). The ratio is that `PATCH` is preferred when you only want to send the "patch document" instead of the whole object (that may be big), so you're sure that you're only affecting the sent fields (but this usually doesn't limit you a subset and this limit is not communicated by the semantic).

In this case the caller can change only the status field so I'd rather go for PUT with a /status URL.

@lucacavallaro what about this?

Reply:
I agree!

Reply:
Both are excellent options, so the one you like best is fine 👍 

If we want to keep the `PATCH` verb and make it more future-proof, we can use the standard `JSON PATCH` https://www.rfc-editor.org/rfc/rfc6902